linera-rpc/src/client.rs

```diff
@@ -13,18 +15,20 @@ use linera_core::{
 
 #[derive(Clone)]
 pub enum Client {
-    Grpc(GrpcClient),
-    Simple(SimpleClient),
+    Grpc(grpc::Client),
```

Comment:
Looks like there is still a bit of refactoring here :)

Reply:
This PR is atop that PR — the refactoring was done to enable this PR.  :)

Reply:
gotcha